enum.js

```diff
@@ -13,50 +22,43 @@ const prototype = {
         return cases[c]()
       }
     }
-  },
-  get [Symbol.toStringTag] () {
-    return `${this.label}.${this.unwrap()}`
-  },
-}
-
-const Enum = function (value, members, label) {
-  me = Object.create(prototype)
+    throw new Error("unreachable")
```

Comment:
lol feels like the failure case for something called `when` should be `"never"`. I don't actually think we should change this, I just think words are funny.